extensions/mapreduce/src/main/resources/benchmark-spark-mr.xml

```diff
@@ -0,0 +1,107 @@
+<!-- RadarGun 3.0 benchmark -->
+<benchmark xmlns="urn:radargun:benchmark:3.0">
+
+   <!-- Specifies where should the master open socket -->
+   <master bindAddress="${master.address:127.0.0.1}" port="${master.port:2103}"/>
+
+   <!-- List of cluster configurations where the benchmark should run -->
+   <clusters>
+      <cluster size="5">
+         <group name="master" size="1" />
```

Comment:
Would it be possible to write a benchmark that would do apple-apple comparison of Spark vs. Infinispan (or generally anything that has MR)?